scripts/scil_decompose_connectivity.py

```diff
@@ -274,8 +276,7 @@ def main():
 
     logging.info('*** Loading streamlines ***')
     time1 = time.time()
-    sft = load_tractogram_with_reference(parser, args, args.in_tractogram,
-                                         bbox_check=False)
+    sft = load_tractogram_with_reference(parser, args, args.in_tractogram)
     sft.remove_invalid_streamlines()
```

Comment:
Here you should remove the invalid only if the check is off (if it is one, the check is done internally).
Skipping it if possible would save time.